### ISSUE TITLE
Some vagrant tests fail

### ISSUE THREAD

**Issue by treydock** [sensu/sensu-puppet#891]
## Description of problem

Several Vagrant boxes fail when testing changes:

```
sensu-server                pass
sensu-server-cluster        pass
sensu-server-puppet5        pass
sensu-server-enterprise     pass
el7-client                  pass
el6-client                  pass
ubuntu1604-client           pass
ubuntu1404-client           pass
amazon201703-client         failed - vagrant
win2012r2-client            pass
win2012r2-client-chocolatey failed - unable to install from chocolatey
debian9-client              failed - vagrant
debian8-client              pass
debian7-client              pass
macos-client                failed - vagrant
```

`failed - vagrant` items failed before box was even accessible via SSH, some failed to even boot and some failed to install drivers necessary to mount module to /vagrant.

## Platform and version information

This was observed while testing #878

**Comment by ghoneycutt** [comment]
Saw an error with macos-client as it is using an old version of stdlib. I could not reproduce any errors with Debian 9 and Amazon Linux. Confirmed Chocolatey is having issues.

**Comment by ghoneycutt** [comment]
@treydock I've got a fix for macos. Could you please look at win2012r2-client-chocolatey.

**Comment by ghoneycutt** [comment]
Released in v2.52.0
